crowdsourcing/viewsets/project.py

```diff
@@ -89,9 +89,9 @@ class ModuleViewSet(viewsets.ModelViewSet):
 
     def get_queryset(self):
         queryset = Module.objects.all()
-        requesterid=self.request.query_params.get('requesterid',None)
```

Comment:
ok this is the ModuleViewset and it will be used in the main feed to show the available tasks out there where people can start working on them, there is mapping between tasks and workers and that's a different serialier/viewset where we query the tasks by worker id, we can query the projects by requester id and collect the existing modules of that project